workshops/facial_expression_recognizer/README.md

```diff
@@ -0,0 +1,155 @@
+```
+- The code above says that:
+  * Import complete `cv2` commonly known as Open CV library. It provides us various computer vision solutions. We will use this library mainly for reading and manipulating images. You can read more about `Open Cv library` [on the OpenCV Docs](https://opencv.org/about/)
+  * Import `FER` function from `fer` library. This is the main library we will be using in this workshop.
+  * Add `matplotlib.pyplot` module as `plt`. It will help us to plot graphs 
+  * Include `matplotlib.image` module as `mpimg` .The matplotlib Library's [matplotlib.image](https://matplotlib.org/tutorials/introductory/images.html) is used to plot image on the graph with `x` and `y` axis.
+  
+  3. Next, we'll predict emotions of Still Images:
+  
+```python
+image = cv2.imread("abc.jpg")
+detector = FER()
+print(detector.detect_emotions(image))
+```
+- Give a try to understand what the code above is trying to say. Here is the explanation of the above code:
+  * Our `image` variable loads an image from specified file in our case `abc.jpg` by using `cv2.imread("abc.jpg")`method. 
+  * Next we will assign `FER()` which was imported earlier to the **detector** variable.
+  * print syntax with `detector.detect_emotions(image)` as parameter will returns dictionary of bounding box notations.
+Below is Sample Output:
+```python
+[{'box': [277, 90, 48, 63], 'emotions': {'angry': 0.02, 'disgust': 0.0, 'fear': 0.05, 'happy': 0.16, 'neutral': 0.09, 'sad': 0.27, 'surprise': 0.41}]
+```
+**NOTE** If you want to add an image in `repl.it` IDE refer to the below Video:
+
+![Upload Image Guide](https://cloud-3jh0ilrgt.vercel.app/0uploadimage.gif)
+
+## What is going on underhood🤔🤔
+
+The `detector` is the main part of our workshop. Let's see what the detector do:
+A detector is an object created to store `FER()`. Detector returns an Ordered Dictionary of Bounding Box Notations. It observes and detects where the face is situated, classifies, and rates its probability with decimal values from `0` to `1` for all 6 emotions respectively. FER exhibit Keras model built with CNN or well know as Convolutional Neural Networks. It stores the values in the `HDF5` model. FER by default detect faces using OpenCV's  Haar Cascade Classifier. Alternatively, we can use a more advanced Multi-Cascade Convolutional Network in short `MTCNN`. It will use Peltarion API in the backend in place of the Keras model. By default FER uses CNN but if want to use advanced MTCNN just pass `mtcnn=True` in FER parentheses just like `detector = FER(mtcnn=True)`. for further use we will use the MTCNN model for more accurate results.
+
+## Data is the new oil!
+
+Data is known as the new oil of the Twenty-First Century. We can do different things with data generated above which we will discuss in the `Hacking Section` at last so stay tuned. We will store the emotion and detected face. Then display bounding yellow box around face detected and scores below it. Let's see below how we can do this.
```

Comment:
Here you have written "we will store the emotion and the detected face" can u please specify "where?" so it will be something like "we will store the emotion and the detected face in the ____ variable"